charts/prometheus-elasticsearch-exporter/values.yaml

```
@@ -250,6 +253,19 @@ serviceMonitor:
   metricRelabelings: []
   sampleLimit: 0
 
+podMonitor:
+  ## If true, a PodMonitor CRD is created for a Prometheus Operator
+  ## https://prometheus-operator.dev/docs/operator/api/#monitoring.coreos.com/v1.PodMonitor
+  ##
+  enabled: false
```

Let us make `namespace` available in the file as well, please.

```suggestion
  enabled: false
  namespace: ""
```